utopian-io/v2.utopian.io
# Feature/work experience/ut 255

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

---

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_No work experience_

* Go to user profile

**Expected result:** The text : "tell us about your different jobs and projects" should display

_Add new work experience_

* Go to user profile
* Input all required fields
* Click Create

**Expected result:** You should see the new work experience added.

_Edit work experience_

* Go to user profile
* Click Edit on an existing work experience

**Expected result:** You should see the edit form displayed and populated with the selected work experience.

_Update work experience_

* Go to user profile
* Click Edit on an existing work experience
* Update fields
* Click Update

**Expected result:** Updated information is displayed on the work experience list; the edit for is collapsed after update

_Delete work experience_

* Go to user profile
* Click Delete on an existing work experience
* Click Yes

**Expected result:** A confirmation dialog will display on click on Delete; work experience should be removed/delete in the work experience list once "Yes" is clicked on the confirm dialog
